src/containers/BooksForm.js

```diff
@@ -10,31 +14,76 @@ const category = [
   { id: 7, name: 'Sci-fi' },
 ];
 
-const BooksForm = () => (
-  <div>
-    <form>
-      <label htmlFor="title">
-        The Title of the Book
-        <input
-          type="text"
-          placeholder="enter the title of the book"
-          name="title"
-          id="title"
-        />
-      </label>
-      <label htmlFor="select">
-        <select>
-          {category.map(c => (
-            <option key={c.id}>{c.name}</option>
-          ))}
-        </select>
-      </label>
-      <button type="submit" value="submit">
-        Submit
-      </button>
-    </form>
-  </div>
-);
-
-
-export default BooksForm;
+class BooksForm extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      title: '',
+      category: '',
+    };
+
+    this.handleChange = this.handleChange.bind(this);
+    this.handleSubmit = this.handleSubmit.bind(this);
+  }
+
+  handleChange = (ev) => {
+    ev.preventDefault();
+    const { value, name } = ev.target;
+    this.setState({ [name]: value });
+  }
+
+  handleSubmit = (ev) => {
+    ev.preventDefault();
+    const { title, category } = this.state;
+    const id = Math.floor(Math.random() * 100);
+    const { createBook } = this.props;
+    createBook({ id, title, category });
+
+    this.setState({
+      title: '',
+      category: '',
+    });
```

Comment:
[Optional] 

Using [`HTMLFormElement.reset()`](https://developer.mozilla.org/en-US/docs/Web/API/HTMLFormElement/reset), we should reset the form inputs too, when a form is submitted and the value is saved, `setState` sets the values to an empty string, but the form still holds such data, submit another form immediately does not yield an intended result. See demo below:

| Current behaviour | Suggested Behaviour |
| :-----: | :-----: |
| ![record1](https://user-images.githubusercontent.com/36057474/83014314-63a98480-a016-11ea-8f6e-4668d8eb37d5.gif)  |  ![record1](https://user-images.githubusercontent.com/36057474/83014585-e7637100-a016-11ea-84c7-c6ba5cceff59.gif) | 


----- 
_This is marked as optional only because the previous reviewer didn't mention of it, I would string recommend you fix this bug._